test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/AbstractCloudIntegrationTest.java

```diff
@@ -98,10 +98,6 @@ public void initializeDeployment() {
     public void cleanEnvironment() {
         if (deploymentScenario != null) {
             deploymentScenario.undeploy();
-
-            if (gitProvider != null) {
-                gitProvider.deleteGitRepository(deploymentScenario.getNamespace());
```

Comment:
Why did you delete this code? Does it cause any issue with Gogs?

Reply:
Yes, it seems that delete method is called twice. At least in some cases. For example in "KieServerS2iJbpmIntegrationTest". It is called here:
- https://github.com/kiegroup/kie-cloud-tests/blob/master/test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/s2i/KieServerS2iJbpmIntegrationTest.java#L103
- https://github.com/kiegroup/kie-cloud-tests/blob/master/test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/AbstractCloudIntegrationTest.java#L103

Reply:
Ok, but why did you delete it here?

Reply:
It's also problem that in this place it is trying to delete wrong git project. Will try to get to know more and decide better. :-)